packages/components/src/components/product-card/index.tsx

```diff
@@ -0,0 +1,89 @@
+import { cn } from '../../cn';
+import { FOUR_MAIN_PRODUCTS, ProductInfo, PRODUCTS } from '../../products';
+import { HighlightDecoration } from '../decorations';
+import { ArrowIcon } from '../icons';
+import { ReactComponent as HiveDecoration } from './hive-decoration.svg';
+import { ReactComponent as HiveGatewayDecoration } from './hive-gateway-decoration.svg';
+import { ReactComponent as MeshDecoration } from './mesh-decoration.svg';
+import { ReactComponent as YogaDecoration } from './yoga-decoration.svg';
+
+const cardDecorations = {
+  [PRODUCTS.HIVE.name]: HiveDecoration,
+  [PRODUCTS.YOGA.name]: YogaDecoration,
+  [PRODUCTS.MESH.name]: MeshDecoration,
+  [PRODUCTS.HIVE_GATEWAY.name]: HiveGatewayDecoration,
+};
+
+export function MainProductCard({ as: Root, product, ...rest }: ProductCardProps) {
+  const Decoration = cardDecorations[product.name];
+  const Icon = product.logo;
+
+  const isHive = product.name === PRODUCTS.HIVE.name;
+
+  return (
+    <Root
+      key={product.name}
```

Comment:
do we need key?

Reply:
No, I copied it and didn't clean up, good catch.